aria/modeling/orchestration.py

```diff
@@ -258,29 +290,17 @@ def execution(cls):
     def inputs(cls):
         return relationship.many_to_many(cls, 'parameter', prefix='inputs', dict_key='name')
 
-    status = Column(Enum(*STATES, name='status'), default=PENDING)
-
-    due_at = Column(DateTime, nullable=False, index=True, default=datetime.utcnow())
-    started_at = Column(DateTime, default=None)
-    ended_at = Column(DateTime, default=None)
+    implementation = Column(String)
     max_attempts = Column(Integer, default=1)
-    retry_count = Column(Integer, default=0)
     retry_interval = Column(Float, default=0)
     ignore_failure = Column(Boolean, default=False)
+    due_at = Column(DateTime, nullable=False, index=True, default=datetime.utcnow())
```

Comment:
if you already do the separation to "state" fields and other, this should probably be under state.